setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php

```diff
@@ -123,4 +124,31 @@ public function validate(InputInterface $input)
 
         return $errors;
     }
+
+    /**
+     * Get admin user data
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return void
+     */
+    protected function setMissingValues(InputInterface &$input, OutputInterface $output)
+    {
+        if (!method_exists($input, 'promptForOption')) {
```

Comment:
Can you clarify the question please? Literally, there should be an interface with the promptForOption method and here you should either have it in ``protected function setMissingValues(InputInteractiveInterface &$input, OutputInterface $output)`` (better) or call ``instanceof`` (still ok).